core/includes/drupal.inc

```diff
@@ -2591,6 +2591,51 @@ function views_clean_css_identifier($identifier, $filter = array(' ' => '-', '/'
   backdrop_clean_css_identifier($identifier, $filter);
 }
 
+/**
+ * Finds the most recently changed nodes that are available to the current user.
+ *
+ * @param $number
+ *   (optional) The maximum number of nodes to find. Defaults to 10.
+ *
+ * @return
+ *   An array of node entities or an empty array if there are no recent nodes
+ *   visible to the current user.
+ */
+function node_get_recent($number = 10) {
```

Comment:
Should we change that variable name to `$limit` instead?

Reply:
No opinion. Not enough experience in naming variables in core, I'll take your opinion.

Reply:
Since we moved an existing function from one place to another, I say we leave it alone. This code is deprecated and will be removed "soon" anyway.